orangewidget/gui.py

```diff
@@ -186,7 +186,7 @@ def __setattr__(self, name, value):
 
 
 def miscellanea(control, box, parent,
-                addToLayout=True, stretch=0, sizePolicy=None, addSpace=None,
+                addToLayout=True, stretch=0, sizePolicy=None,
```

Comment:
Let us pray that nobody called this function with non-keyword arguments (which is indeed very unlikely as this is a very internal function, although not named as such).

Maybe add a `*, ` before `addToLayout` to ensure that the function is called wiith keyword arguments and that any code that may currently call it positional arguments will fail here and not silently misbehave latter.

Reply:
👍